db/migrations/versions/b2f22185655c_.py

```diff
@@ -0,0 +1,46 @@
+"""empty message
+
+Revision ID: b2f22185655c
+Revises: 755b1be91ed5
+Create Date: 2022-09-16 18:35:47.922378
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "b2f22185655c"
+down_revision = "755b1be91ed5"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("applications", schema=None) as batch_op:
+        batch_op.add_column(sa.Column("key", sa.String(), nullable=False))
+        batch_op.add_column(
+            sa.Column("reference", sa.String(), nullable=False)
```

Comment:
I think we may need to make these Nullable else they will break for any existing applications in the DB which won't have those fields set. Have we tested this migration on the docker-runner?

Reply:
Yep hadn't got to the migrations yet. I think these should probably be not null fields after migration so I've added server_defaults during migration (for existing rows) and then removed the server_default straight after